app/main.py

```
@@ -640,12 +642,26 @@ async def unified_chat_stream(model: str, messages: list, system: str | list = N
+        if not raw_verification:
+            return "ALL_CLAIMS_VERIFIED"
+        
+        # Filter out false-alarm DISPUTED lines regarding static form links
+        lines = [line.strip() for line in raw_verification.split("\n") if line.strip()]
+        filtered_lines = []
+        for line in lines:
+            if "DISPUTED:" in line and any(kw in line.lower() for kw in ("/public/docs/forms", "official form", "grievance form", "grievance document", "provided links")):
+                continue
+            filtered_lines.append(line)
+
+        if not filtered_lines or all("VERIFIED:" in l or l == "ALL_CLAIMS_VERIFIED" for l in filtered_lines):
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)